include/librealsense2/h/rs_context.h

```diff
@@ -40,6 +40,7 @@ rs2_context* rs2_create_context(int api_version, rs2_error** error);
 *             full: provide all conversions (e.g., YUYV -> RGB8 etc.)
 *             basic: use mostly raw camera formats (no RGB8 etc.); convert interleaved (Y8I -> 2xY8)
 *             raw: leave all formats from camera as they are
+*         options-update-interval      - (uint32_t) time interval in milliseconds for option value change notifications
```

Comment:
Maybe add `(see rs2_set_options_changed_callback)`

Reply:
Also, need to add the default

Reply:
Added